l1-contracts/src/core/libraries/validator-selection/ValidatorSelectionLib.sol

```diff
@@ -114,50 +120,61 @@ library ValidatorSelectionLib {
       return;
     }
 
-    uint256 needed = committee.length * 2 / 3 + 1;
-    require(
-      _signatures.length >= needed,
-      Errors.ValidatorSelection__InsufficientAttestationsProvided(needed, _signatures.length)
-    );
-
     // Validate the attestations
+    uint256 needed = committeeSize * 2 / 3 + 1;
     uint256 validAttestations = 0;
 
+    address[] memory reconstructedCommittee = new address[](committeeSize);
+
     bytes32 digest = _digest.toEthSignedMessageHash();
-    for (uint256 i = 0; i < _signatures.length; i++) {
+    for (uint256 i = 0; i < _attestations.length; i++) {
       // To avoid stack too deep errors
-      Signature memory signature = _signatures[i];
-      if (signature.isEmpty) {
-        continue;
+      CommitteeAttestation memory attestation = _attestations[i];
+      // TODO(https://github.com/AztecProtocol/aztec-packages/issues/14283): as part of bitmap / storage optimisation, this check will change to whatever the bitmap includes
```

Comment:
there is a not in https://github.com/AztecProtocol/aztec-packages/issues/7978 at the bottom about it and an issue already 👀 https://github.com/AztecProtocol/aztec-packages/issues/10455